src/main/scala/uk/gov/homeoffice/rtemailer/govnotify/GovNotifyClientWrapper.scala

```diff
@@ -1,33 +1,61 @@
 package uk.gov.homeoffice.rtemailer.govnotify
 
 import cats.effect._
+import cats.data.EitherT
 import com.typesafe.scalalogging.StrictLogging
 import uk.gov.service.notify.{NotificationClient, Template, TemplatePreview, SendEmailResponse}
 import uk.gov.homeoffice.rtemailer.model._
 import uk.gov.homeoffice.domain.core.email.Email
 import scala.collection.JavaConverters._
 import scala.util.Try
 
+case class TemplateWC(template :Template, client :NotificationClient) {
+  def getPersonalisation() :java.util.Optional[java.util.Map[String,Object]] = template.getPersonalisation()
+  def getName() :String = template.getName()
+  def getId() :java.util.UUID = template.getId()
+}
+
 class GovNotifyClientWrapper(implicit appContext :AppContext) extends StrictLogging {
   import uk.gov.homeoffice.rtemailer.Util._
 
-  lazy val notifyClient = new NotificationClient(appContext.config.getString("govNotify.apiKey"))
+  val notifyClient1 = new NotificationClient(appContext.config.getString("govNotify.apiKey"))
```

Comment:
Does 1 or 2 represent anything more specific ? like client or something . I think it would be good change from 1 and 2 to more specific name if possible.

Reply:
The system treats them exactly the same. it doesn't know, or care, that 1 = global entry and 2 = RT, which is why I didn't name them like this. I would have used a list but the syntax in config files is a bit too fiddly so I thought this would be easy enough.